batch-job.go

```diff
@@ -205,7 +205,7 @@ type GenerateBatchJobOpts struct {
 
 // GenerateBatchJob creates a new job template from standard template
 // TODO: allow configuring yaml values
-func (adm *AdminClient) GenerateBatchJob(ctx context.Context, opts GenerateBatchJobOpts) (string, error) {
+func (adm *AdminClient) GenerateBatchJob(opts GenerateBatchJobOpts) (string, error) {
```

Comment:
unrelated change please revert - pass `_ context.Context` like this, we need this for future purposes.